checks/apps/sirius/sirius_check.py

```diff
@@ -0,0 +1,185 @@
+import json
+import pathlib
+import re
+import sys
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.osext as osext
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
+from extra_launcher_options import ExtraLauncherOptionsMixin
+from sarus_extra_launcher_options import SarusExtraLauncherOptionsMixin
+from cuda_visible_devices_all import CudaVisibleDevicesAllMixin
+
+class sirius_scf_base_test(rfm.RunOnlyRegressionTest):
+    test_folder = parameter(['Si63Ge'])
+    executable = 'sirius.scf'
+    strict_check = False
+    use_multithreading = False
+    maintainers = ['antonk']
+    data_ref = 'output_ref.json'
+    fout = 'output.json'
+
+    def __init__(self, num_tasks):
```

Comment:
The init is not needed, you can set directly the:
`num_tasks` and `executable_opts` at the child class

Reply:
fixed